security/basic/src/main/java/io/quarkus/ts/openshift/security/basic/SecurityFilter.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.ts.openshift.security.basic;
+
+import jakarta.inject.Singleton;
+
+import io.quarkus.vertx.web.RouteFilter;
+import io.vertx.ext.web.RoutingContext;
+
+@Singleton
+public class SecurityFilter {
```

Comment:
Please add explanation, why is the filter here (ideally link the issue + comment that issue only appears if security filter is here).
There is a piece of code, that does nothing on a first glance. It would be really good to have an explanation why is it necessary.